docs/documents.js

```diff
@@ -290,6 +297,13 @@ var documents = [
 						url: './packages/snap-store-mobx/src/Abstract/README.md',
 						searchable: true,
 					},
+					{
+						label: 'Meta',
+						route: '/package-storeMobx-meta',
+						type: 'markdown',
+						url: './packages/snap-store-mobx/src/Meta/README.md',
+						searchable: true,
+					},
```

Comment:
Can we relocate this in the order?

Maybe under 'search' or 'cart' or 'storage'?
<img width="248" alt="image" src="https://github.com/searchspring/snap/assets/5060400/2da6f987-c16f-4f91-8a08-cec9c2d4250b">